### PR TITLE
Week 3 step 5

### PR DESCRIPTION
**Pull request by SeanT18**
I will be able to store comments permanently so they can be seen whenever they're uploaded.

### CODE REVIEW

**Comment by sidharthp** [comment]
This has a bunch of overlap with [1]. Could you perhaps change the base so that it only accounts for new changes? You can probably use the feature [2].

[1] https://github.com/SeanT18/step/pull/8
[2] https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request

**Comment by SeanT18** [comment]
> This has a bunch of overlap with [1]. Could you perhaps change the base so that it only accounts for new changes? You can probably use the feature [2].
> 
> [1] #8
> [2] https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request

Would you like me to change bases with this step and the previous?

**Comment by yjennywang** [comment]
I see sidharth just approved the previous one, so if you merge that PR, I can just start reviewing this one! :) No need for any action then.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (yjennywang) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: consider setting "comments" as a constant at class-level:

e.g. public static final String TASK_COMMENT_PROPERTY = "comment";

Since you will also need to reference it to get the comments.

**Comment by SeanT18** [comment]
> I see sidharth just approved the previous one, so if you merge that PR, I can just start reviewing this one! :) No need for any action then.

I'm still waiting for his approval. But I think he meant I can change bases on the previous branch from master to this branch?

**Review by yjennywang — APPROVED** [review]
One small comment, but otherwise, looks good to me.

I started reviewing from 26fb647ff31ad401219e9d4a9d9021fb158a0f35 which I believe is where the last pull review left off from.

You may need to do a git pull --rebase master (after merging the last pr) before you can merge this change; let me know if you need any help.

**Comment by yjennywang** [comment]
> > I see sidharth just approved the previous one, so if you merge that PR, I can just start reviewing this one! :) No need for any action then.
> 
> I'm still waiting for his approval. But I think he meant I can change bases on the previous branch from master to this branch?

Ah, my mistake! I thought the other PR got approved already, sorry.

I manually looked at the commits that were added to this branch (which weren't on the others). So feel free to merge this PR after the last PR is approved and merged.

**Comment by SeanT18** [comment]
I had some more things that I just pushed to the branch can you see them even though you already approved?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (SeanT18) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: I got a lot of errors when I tried to add that and replace my "comments" with it.

**Comment by yjennywang** [comment]
I can see it! Will take a look and then re-approve.

**Pull request merged**
